src/commands/run.js

```diff
@@ -9,14 +9,19 @@ module.exports = {
 	builder: yargs => yargs,
 	handler: argv => {
 		const { NODE_ENV } = process.env;
+		const envString = NODE_ENV || 'development';
 		console.log(
-			chalk.green(`NODE_ENV=${process.env.NODE_ENV || '(empty)'}`)
+			chalk.green(`NODE_ENV=${envString}`)
 		);
-		if (!NODE_ENV !== 'production') {
-			// TODO: check for prerequisites (e.g. up-to-date build/... files)
-			// execute the start-dev script
-			console.log(chalk.blue('Running the dev app server...'));
-			startDev();
+		if (envString !== 'development') {
+			console.log(chalk.red(`Cannot run dev server with NODE_ENV=${envString}`))
+			console.log(chalk.red(`Set NODE_ENV=development`))
```

Comment:
After thinking it over, I think its best I take a harder stance on this. While I agree that its often best to be explicit, its a bit different with env vars. We should only set env vars as a last result, such as with secrets. We should also be unambiguous with what is required of a dev environment. I stand by my original comment.